_data/pages.yaml

```diff
@@ -27,51 +61,203 @@
                   id: "see-also-section"
             - title: "Web-ext Webpack plug-in"
               url: "/documentation/develop/web-ext-webpack-plug-in/"
-            - title: "Web-ext command reference"
-              url: "/documentation/develop/web-ext-command-reference"
+            - title: "web-ext command reference"
+              url: "/documentation/develop/web-ext-command-reference/"
+              subpageitems: 
+                - title: "Commands"
+                  id: "commands"
+                - title: "Global options"
+                  id: "global-options"
+                - title: "Setting option environment variables"
+                  id: "setting-option-environment-variables"
+                - title: "See also"
+                  id: "see-also"
             - title: "Browser API Polyfill"
-              url: "/documentation/develop/browser-api-polyfill"
+              url: "/documentation/develop/browser-api-polyfill/"
             - title: "Extensions and the Add-on ID"
-              url: "/documentation/develop/extensions-and-the-add-on-id"
+              url: "/documentation/develop/extensions-and-the-add-on-id/"
+              subpageitems:
+                - title: "Basic workflow with no add-on ID"
+                  id: "basic-workflow-with-no-add-on-id"
+                - title: "When do you need an add-on ID?"
+                  id: "when-do-you-need-an-add-on-id"
             - title: "Request the right permissions"
-              url: "/documentation/develop/request-the-right-permissions"
+              url: "/documentation/develop/request-the-right-permissions/"
+              subpageitems: 
+                - title: "Introduction"
+                  id: "introduction"
+                - title: "Advised permissions"
+                  id: "advised-permissions"
+                - title: "Avoid unnecessary permissions"
+                  id: "avoid-unnecessary-permissions"
+                - title: "Request permissions at runtime"
+                  id: "request-permissions-at-runtime"
+                - title: "Add information about permissions to your extensions AMO page"
+                  id: "add-information-about-permissions-to-your-extensions-amo-page"
         - category: "User Experience"
           pages:
             - title: "Build a secure extension"
               url: "/documentation/develop/build-a-secure-extension/"
```

Comment:
This seems to be linking to accessibility documentation and the accessibility page is a 404 from the sidebar suggesting that the generation might not be quite right.

Reply:
How was this resolved? Please add a comment so it's clear what action was taken.

Reply:
Ah, sorry! This page had the wrong front matter title and permalink, which I fixed.